src/nginx/t/ServiceControl.pm

```diff
@@ -211,13 +211,9 @@ sub gen_report_body {
   my @metrics = (
     gen_metric_int64(
       'serviceruntime.googleapis.com/api/producer/request_count', 1),
```

Comment:
src/nginx/t/by_consumer_metrics.t has the test with cunsumer_project_id: 1002409420961

https://github.com/cloudendpoints/esp/pull/235/files#diff-4301c1e4ab85eca9298cb213cb5c5af7 has